docs/command/mongocli-config-rename.txt

```diff
@@ -19,7 +19,26 @@ Syntax
 
 .. code-block::
 
-   mongocli config rename <oldName> <newName> [options]
+   mongocli config rename <oldProfileName> <newProfileName> [options]
+
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - oldProfileName
+     - string
+     - true
+     - Name of the profile to rename.   * - newProfileName
+     - string
+     - true
+     - New name of the profile.
```

Comment:
@gssbzn Looks multiple arguments are being formatted incorrectly. (`* - newProfileName` should be inserted on the next line). I'm hoping this is a quick fix in `cobra2snooty`?

Reply:
yep, PR here https://github.com/mongodb-labs/cobra2snooty/pull/4

Reply:
updating with master and re running should fix this now 🤞